src/event-tracker.ts

```diff
@@ -57,7 +56,7 @@ export class EventTracker {
 			if (unmanaged || mixpanelHost == null) {
 				return;
 			}
-			this.client = Mixpanel.init(mixpanelToken, {
+			this.client = mixpanel.init(mixpanelToken as string, {
```

Comment:
This cast shouldn't be required, the `ConfigType<'mixpanelToken'>` should resolve to a string.

Reply:
it's `string | undefined`

Reply:
I'd prefer we check for thay here then https://github.com/balena-io/balena-supervisor/pull/1106/files#diff-aaf08893d3e6fd02df6a69b14f239847R56